pkgs/by-name/pe/peazip/package.nix

```
@@ -33,14 +34,15 @@ stdenv.mkDerivation rec {
   '';
 
   nativeBuildInputs = [
-    libsForQt5.wrapQtAppsHook
+    qt6.wrapQtAppsHook
     lazarus
     fpc
   ];
 
   buildInputs = [
     xorg.libX11
-    libsForQt5.libqtpas
+    qt6.qtbase
```

Could be in a:

```nix
] ++ (with qt6Packages; [
```

…together with the `libqtpas` to save the repeated `qt6Packages`, but:

```suggestion
    qt6Packages.qtbase
```

…is fine.